examples/202-NN_examples.ipynb

```diff
@@ -33,7 +33,8 @@
     "    * [N-BEATS Model](#section_3_9)\n",
```

Comment:
<p>Shouldn't we add a plot with a result? We could also add plot for best chronos model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='152'/>

Reply:
I added one plot for each model type. I think it is not so important to show plot for each run.

Reply:
For chronos model we plotted a bad result)